docs/api/session.md

```diff
@@ -91,6 +91,43 @@ session.defaultSession.on('will-download', (event, item, webContents) => {
 })
 ```
 
+#### Event: 'extension-loaded'
+
+Returns:
+
+* `event` Event
+* `extension` [Extension](structures/extension.md)
+
+Emitted after an extension is loaded. This occurs whenever an extension is
+added to the "enabled" set of extensions. This includes:
+- Extensions being loaded from `Session.loadExtension`.
+- Extensions being reloaded:
+   * if the extension requested it ([`chrome.runtime.reload()`](https://developer.chrome.com/extensions/runtime#method-reload)).
+
+<!--
+Refer to //extensions/browser/extension_registry_observer.h for further
+explaination.
+-->
```

Comment:
Perhaps include this in the visible part of the documentation?

Or, better, fully enumerate the situations that can cause this event in Electron.

Reply:
I've updated it to include more info, but the parts about enabled/disabled extensions doesn't make a lot of sense until #25386 is merged. However, enabled/disabled extensions still are technically part of the extension system implementation regardless.

Reply:
I'll hold off on mentioning the concept of enabled/disabled extensions for now. I'll plan to include them in the mentioned PR.